l10n_es_aeat/readme/CONTRIBUTORS.rst

```diff
@@ -16,3 +16,7 @@
 * Valentin Vinagre <valentin.vinagre@sygel.es>
 * Manuel Regidor <manuel.regidor@sygel.es>
 * Jairo Llopis (https://www.moduon.team)
+* Emilio Pascual (https://www.moduon.team)
```

Comment:
Añadir 4 contribuidores para un simple valor por defecto no lo veo razonable.

Reply:
Es un trabajo en equipo, @loida-vm y @ArantxaSudon han detectado la necesidad y posteriormente han probado la solución, @rafaelbn ha diseñado conmigo la solución y yo la he desarrollado en código. 
No entiendo porque no es razonable poner a las personas que han intervenido, yo lo veo justo.

Reply:
Estoy de acuerdo con @etobella No es una buena práctica ponerse como autor o como contribuidor si no has hecho una aportación significativa al módulo. Parecería entonces que todos sois responsables de haber desarrollado el módulo, lo cual no es para nada así. Entiendo que desde OCA debería haber mejor directrices al respecto. Propondré hablarlo en el board de OCA.

Reply:
@pedrobaeza @JordiBForgeFlow @etobella , os entiendo perfectamente y no comparto vuestro opinión con ninguno. La cuestión es que esto no son 12 líneas salidas de la nada, este y otros tantos PRs, son todos el resultado de un trabajo en equipo de 3 semanas y que vamos poco proponiendo por piezas tal como se debe hacer, pero cada una de esas piezas no tiene una singular y aislada contribución si no es que un gran trabajo y esfuerzo que tenemos que dividir en cachitos por la modularidad que se ha de tener técnicamente.

¿Entendéis?

This is a topic I also would like to discuss with @vdewulf and the functional team. A large 4-week effort of 4 functional consultants (tax experts) doing tests, collecting user feedback, etc. **must be divided into dozens of mini public contributions** to give the work done and perfectly written to 1 developer and then you, the technical, out of context and without empathy then **consider that this pile of mini contributions does not deserve any type of recognition**.

The board should consider as I said the things that have never considered. 👍🏼

Reply:
Please show where the "dozens of contributions" on **this module** are, and the percentage of code that they mean in the total of the module.

Reply:
@pedrobaeza El problema que veo es que da lo mismo si has creado el modulo y has dedicado 4 meses, y otro ha puesto 5 minutos en hacer un fix. Ahora mismo tienen el mismo peso los dos, como contribuidores. Hasta se podria automatizar. Para cada commit que haces, te ponen como contribuidor. Entonces, tienes una lista de 30 contribuidores por módulo. Yo veo que nunca se ha seguido tal regla. Hay unas reglas no escritas desde siempre que establecen las buenas prácticas. La gente las sigue, sin más.

Reply:
Correcto, y en contribuciones muy pequeñas que yo he hecho en muchos módulos, ni me he molestado en añadirme en contribuidores. Aquí no es solo que se quiera añadir un contribuidor por una aportación pequeña. ¡Es que se quieren añadir 4! Y además, repitiendo en cada línea la web de la empresa.

Reply:
[...]

- https://github.com/OCA/l10n-spain/pull/3409
- https://github.com/OCA/l10n-spain/issues/3347
- https://github.com/OCA/l10n-spain/pull/3359
- https://github.com/OCA/l10n-spain/pull/3380
- https://github.com/OCA/l10n-spain/pull/3389
- https://github.com/OCA/l10n-spain/pull/3395
- https://github.com/OCA/l10n-spain/pull/3397
- https://github.com/OCA/l10n-spain/pull/3399
- https://github.com/OCA/l10n-spain/pull/3402
- https://github.com/OCA/l10n-spain/pull/3086

...and those that remain to come from model AEAT 180 and AEAT 190, although here we and many other contributors who have left and will never return are discouraged.

What I see is Spanish PSCs without any single PR (or review) in one full year or review, PSC that merge his own PRs without any review at all (this, I don't mind), ### taking more care about if 4 great accountant and fiscal consultant are added to an improvement. **Because recognition seems to be more important than the community itself and its growth**.

And myself, with enormous knowledge of tax and AEAT that you could take advantage of, I am rejected as a PSC (for personal reasons) but also now do you want to detract from the efforts of my functional accounting team? This is not important to us, it is important to the OCA and being a larger or smaller community. And I refer to the facts that it has not grown in 8 years, at least.

The reality of the OCA contributions and plurality is to have the ability to empathize with everyone and invite everyone to contribute, recognizing the efforts that are not seen. The @OCA/board should decide yes.

Reply:
As said, most of them are not contributions to **THIS MODULE**, and you have already added contributors on each module, so this is not justified.

Reply:
Of the several PR you are talking, only 3 reference to this module....

They are:

- this one
-  a change of sort (https://github.com/OCA/l10n-spain/pull/3397/files) 
- a PR done by another member that it is not included in your change of contributors list and was done 6 months ago... (#3086)

We are not talking about collaboration and plurality, it was just an statement of a PSC about adding too many people as contributor for a small conributions. Contribution is always welcomed but he stated that it was not necessary to add 4 people as Contributors for such a small change.

Reply:
@EmilioPascual leave @ArantxaSudon or @loida-vm as contributors, just one. Then we can close this discussion easily and not wasting more energy. All happy ❤️ !

Reply:
keep calm, it's already changed